client/src/templates/layouts/NavigationLayout.js

```diff
@@ -622,12 +623,12 @@ const NavigationLayout = (props) => {
     </div>
   );
 
-  if (isAuthenticated) { 
-    tagManagerArgs['dataLayer'] = {userId: user.id};
+  if (isAuthenticated) {
+    tagManagerArgs["dataLayer"] = { userId: user.id };
     TagManager.initialize(tagManagerArgs);
-  } 
+  }
 
-  return <>{renderNavigationBar()}</>; 
+  return <>{renderNavigationBar()}</>;
```

Comment:
This is most likely going to have merge conflicts / looks like you might've started this branch off master instead of rc/v1.1. But yeah please undo the changes besides Line 306.

Reply:
No, this was branched from rc/v1.1.

```
44b8731e (HEAD -> feature/fix-post-alignment-and-background, origin/feature/fix-post-alignment-and-background) Tweak PostCard padding
22ee0ebc Change PostCard margin to padding, remove post title side margin & padding
409b6c97 (origin/rc/v1.1, rc/v1.1) add scroll to auto in create post (#1216)
```